packages/app/pages/course/[cid]/today.tsx

```diff
@@ -72,6 +73,9 @@ export default function Today(): ReactElement {
   const role = useRoleInCourse(Number(cid));
   const { course, mutateCourse } = useCourse(Number(cid));
 
+  const parts = course?.queues && partition(course?.queues, (q) => q.isOpen);
+  const sortedQueues = parts && parts[0].concat(parts[1]); // put open queues first
```

Comment:
hehe, what if we used the vanilla Array `sort` method with a comparator. so smth like this (assuming you want open queues before the non open ones)
```suggestion
  const sortedQueues = [...course?.queues].sort(({isOpen: a}, {isOpen:b}) => Number(b) - Number(a)); // put open queues first
```

Reply:
_take my black magic_

Reply:
ok i had a `sort` first but then i changed it for some reason thinking this was simpler lol. ill put it back 👍